current-federal-non-dotgov.csv

```diff
@@ -399,7 +399,6 @@ fakecheck.us,Federal Agency - Executive,United States Postal Service,,Washington
 faqusps.com,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 fastmaildelivery.net,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 fastmaildelivery.org,Federal Agency - Executive,United States Postal Service,,Washington,DC,
-fdastudios.com,Federal Agency - Executive,Department of Health and Human Services,,Washington,DC,
 fdastudios.net,Federal Agency - Executive,Department of Health and Human Services,,Washington,DC,
 fdastudios.org,Federal Agency - Executive,Department of Health and Human Services,,Washington,DC,
 fdastudios.us,Federal Agency - Executive,Department of Health and Human Services,,Washington,DC,
```

Comment:
What about these other three similar domains?
* fdastudios.net
* fdastudios.org
* fdastudios.us

Are those still claimed by HHS?

Reply:
Hey @dav3r  I was thinking the same thing as well. They don't appear to be active domains. The stakeholder only requested the removal of the .com but I did a check and those domains appear to not belong to the agency either.

Reply:
Excellent, I saw https://github.com/cisagov/scan-target-data/pull/104/commits/b821fe5abf54c44be4d3a47b2f972da2fa1d5e60 and it looks good. 👍